src/csvcubed/readers/cubeconfig/v1_0/templates/mixed-period.json

```diff
@@ -0,0 +1,7 @@
+{
+    "type": "dimension",
+    "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+    "label": "Mixed Time Period",
+    "description": "Template allowing the use of different time periods within one dimension, e.g. 2020-Q4 (quarter), 2021-06 (month), 2019-08-06 (day)",
```

Comment:
The `description` describes the dimension, not the template. So I'd reword it if you could.

Reply:
Thanks, amended